compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -234,7 +231,7 @@ protected Config() throws IOException {
                 new ObjCBlockPlugin(),
                 new AnnotationImplPlugin(),
                 new LambdaPlugin(),         
-                new ShadowFramePlugin()
+                new DebugInformationPlugin()
```

Comment:
Ah, I see it.  So the only extra overhead is really the extra files with the debug data and when this gets processed at unwinding?